### PR TITLE
Edited React example to use Meteor 1.6.1, withTracker, React 15/16

### PR DESCRIPTION
**Pull request by mikkelking**
Issue: https://github.com/VeliovGroup/Meteor-Files/issues/572

### CODE REVIEW

File: docs/react-example.md [hunk 1]
```diff
@@ -1,51 +1,60 @@
...
+## Assumptions
+ - You have Meteor methods for `RemoveFile` and `RemoveFile`
+ - You have a publication called `files.all` 
+ which is a FilesCollection, declared something like this:
 
-## FileUpload.jsx:
+```js
+import { FilesCollection } from 'meteor/ostrio:files'
+
+export const ReportImages = new FilesCollection({collectionName: 'reportimages'});
```
[the remark this answers is not in the document]
Comment (mikkelking) [reply docs/react-example.md]: This is a left over (the name of the collection I use), so I have changed it to `UserFiles` now - which is how you refer to it in the example code. 

I think the capitalized variable names is also used for collection names.

File: docs/react-example.md [hunk 2]
```diff
@@ -1,51 +1,60 @@
...
+
+## FileUpload.js:
 
 ```jsx
-import { ReactMeteorData } from 'meteor/react-meteor-data';
-import React from 'react';
-import { Meteor } from 'meteor/meteor';
-import IndividualFile from './FileIndividualFile.jsx';
-import { _ } from 'meteor/underscore';
+import { withTracker } from 'meteor/react-meteor-data'
```
[the remark this answers is not in the document]
Comment (mikkelking) [reply docs/react-example.md]: Sure, if that's what you prefer.

File: docs/react-example.md [hunk 3]
```diff
@@ -124,21 +133,21 @@ const FileUploadComponent = React.createClass({
...
       // Run through each file that the user has stored
       // (make sure the subscription only sends files owned by this user)
       let display = fileCursors.map((aFile, key) => {
-        // console.log('A file: ', aFile.link(), aFile.get('name'));
-        let link = UserFiles.findOne({_id: aFile._id}).link();  //The "view/download" link
+        // console.log('A file: ', aFile.link(), aFile.get('name'))
+        let link = UserFiles.findOne({_id: aFile._id}).link()  //The "view/download" link
 
-        // Send out components that show details of each file
+        // Send out components that show details of each fil e
```
[the remark this answers is not in the document]
Comment (mikkelking) [reply docs/react-example.md]: fixed

**Comment by mikkelking** [comment]
Changes made as requested

**Comment by dr-dimitru** [comment]
@mikkelking thank you a lot for contribution  🎉👍
Docs are updated, released as [`v1.9.10`](https://github.com/VeliovGroup/Meteor-Files/releases/tag/1.9.10).


Please, support this project by:
 - Star on [GitHub](https://github.com/VeliovGroup/Meteor-Files)
 - Star on [Atmosphere](https://atmospherejs.com/ostrio/files)
 - Check [other packages](https://github.com/search?o=desc&q=topic%3Ameteor-package+org%3AVeliovGroup&s=stars&type=Repositories) we have made
 - Share on [Facebook](https://www.facebook.com/sharer.php?u=https%3A%2F%2Fgithub.com%2FVeliovGroup%2FMeteor-Files) and [Twitter](https://twitter.com/share?url=https%3A%2F%2Fgithub.com%2FVeliovGroup%2FMeteor-Files)
 - [Donate via PayPal](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=FVDSXRFW9VGA2)

**Pull request merged**